Challenge2/src/agent.spec.ts

```diff
@@ -50,16 +60,15 @@ describe("Uni V3 Swap Detector Test Suite", () => {
     mockRealPoolAddress = (
       await getRealPoolAddress(mockUniFactoryAddress, mockInitCodeHash, mockPoolValues)
     ).toLowerCase();
-
-    mockSwapEventArgs = [
-      mockSender,
-      mockRealPoolAddress,
-      ethers.BigNumber.from("-1000000000000000000"), // mock amount0 in wei (assume 18 decimals), so essentially this is -1
-      ethers.BigNumber.from("3000000000000000000000"), // mock amount1 in wei (assume 18 decimals), so essentially this is 3000
-      ethers.BigNumber.from("39614081257132168796771975168"), // mock sqrtPriceX96 is (square root of ratio) * 2^96
-      ethers.BigNumber.from("1000000000000000000000000"), // mock liquidity in wei (assume 18 decimals), so essentially this is 1000
-      40943, // tick = log(3000) / log(1.0001) based on price ratio 3000:1
-    ];
+    mockSwapEventArgs = {
```

Comment:
```suggestion
    mockSwapEventArgs: SwapEventArgs = {
```
Comment to apply the type.

Reply:
Done, I had to use different syntax:

mockSwapEventArgs = { ... } as SwapEventArgs;